README.md

```diff
@@ -463,6 +463,34 @@ The final step once your app is setup is to list in the HubSpot marketplace. You
     No. Only users who have necessary permissions can install and uninstall an app. These permissions can be found in HubSpot portal settings page in the "Users & Teams" tab. </p>
 </details>
 
+<details>
+  <summary>Can I place a call from a custom object?</summary>
+  <p>Yes, calling integrations can now place calls from custom objects as long as they only use the SDK to create the call. In order to take advantage of these improvements, each integration will need to verify that they only use the Calling SDK to create calls and to notify HubSpot in the outgoingCall event.</p>
+
+  <p>1. Verify that the integration is using the Calling SDK to create engagements in the outgoingCall event:</p>
+
+  ```js
+createEngagement: true
```

Comment:
I would include the method in this example, just so they know what we're referring to here.
```
outgoingCall({ createEngagement: true })
```